Source/MonoGame.Extended/Particles/Modifiers/Interpolators/ColorInterpolator.cs

```diff
@@ -18,7 +18,7 @@ public sealed class ColorInterpolator : IInterpolator
         public HslColor FinalColor { get; set; }
         
         public unsafe void Update(float amount, Particle* particle) {
-            particle->Color = HslColor.Lerp(InitialColor, FinalColor, particle->Age);
+            particle->Color = HslColor.Lerp(InitialColor, FinalColor, amount);
```

Comment:
Because the age should only be used in the `AgeModifier`, using amount makes it so that `VelocityModifier` (and maybe future classes like that) also work.